--- authenticate.c	2020-07-06 09:05:46.000000000 +0200
+++ authenticate.c.new	2020-09-25 22:43:03.457662400 +0200
@@ -187,7 +187,7 @@
 			rsyserr(FERROR, errno, "could not open password file %s", filename);
 			exit_cleanup(RERR_SYNTAX);
 		}
-
+/*
 		if (do_stat(filename, &st) == -1) {
 			rsyserr(FERROR, errno, "stat(%s)", filename);
 			exit_cleanup(RERR_SYNTAX);
@@ -196,6 +196,7 @@
 			rprintf(FERROR, "ERROR: password file must not be other-accessible\n");
 			exit_cleanup(RERR_SYNTAX);
 		}
+*/
 		if (MY_UID() == ROOT_UID && st.st_uid != ROOT_UID) {
 			rprintf(FERROR, "ERROR: password file must be owned by root when running as root\n");
 			exit_cleanup(RERR_SYNTAX);